### ISSUE TITLE
Free creative mode flux capacitors ;)

### ISSUE THREAD

**Issue by magnusjjj** [Draco18s/Artifacts#16]
The recharging effect makes all kinds of flux capacitors from thermal expansion into creative mode flux capacitors :)

**Comment by Draco18s** [comment]
Duplicate of #14 
Known issue, but the problem is not with Artifacts.

**Comment by magnusjjj** [comment]
Uhm, no. I think you misunderstand. That API, setdamage? Just because it says 'set damage', does not mean thats what that is for. That (internal minecraft api) is painstakingly *deobfuscated* manually by devs. What it actually *does* is just change the metadatatag regardless. There is no such thing as saying 'i am repairable', short of seeing if you can combine two of the items in an anvil.

Saying you won't fix it is fine (nobody is forcing you to do work, and hey, the mod is awesome and fun so w/e), but the api name is wrong because its *named* wrong :P.  Do you want a patch for it if i write it? The 'does two items combine in an anvil' test is probably the most correct one, and i could probably code that :)

**Comment by Draco18s** [comment]
There *is* an "boolean isRepairable()" function.  And it should be in use here checking that items *are allowed to be repaired* before the effect runs on them.

**Comment by magnusjjj** [comment]
But isnt: https://github.com/Draco18s/Artifacts/blob/master/main/java/com/draco18s/artifacts/components/ComponentRepair.java

**Comment by Draco18s** [comment]
That component is "repair myself."  It doesn't need to check.
https://github.com/Draco18s/Artifacts/blob/master/main/java/com/draco18s/artifacts/components/ComponentRepairOther.java

**Comment by magnusjjj** [comment]
That doesnt check that either :). It only checks isdamaged

**Comment by Draco18s** [comment]
You're right, it doesn't.  It's been too long since I looked at it, as I don't maintain the code itself any more.

Tmtravlr, ball's in your court now. :)
